test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -713,10 +732,11 @@ static void alg_test(odp_crypto_op_t op,
 		     crypto_test_reference_t *ref,
 		     uint32_t digest_offset,
 		     iv_test_mode_t iv_mode,
-		     odp_bool_t bit_mode)
+		     odp_bool_t cipher_bit_mode,
+		     odp_bool_t auth_bit_mode)
 {
 	unsigned int initial_num_failures = CU_get_number_of_failures();
-	const uint32_t reflength = bit_mode ? (ref->length + 7) / 8 : ref->length;
+	const uint32_t reflength = ref->length_is_in_bits ? (ref->length + 7) / 8 : ref->length;
```

Comment:
The same bits to bytes calculation repeats quite a lot, a helper macro for the operation could be useful.